### DISCUSSION TITLE
How to disable the `bash -c vim -- thefile` log after I exit lazygit if I edit/open a file in the lazygit session

### DISCUSSION THREAD

**Discussion by c02y** [jesseduffield/lazygit#2602] [Q&A]
https://user-images.githubusercontent.com/2135996/236550483-0c028220-cb5f-4520-bd01-b16c43e12511.mp4

**Comment by jesseduffield** [comment]
Good question. That appears because we're printing to stdout when running a subprocess. We could find some other place to write to instead. Feel free to raise an issue for this
